### PR TITLE
 Updated certification articles

### LINKED ISSUE #535: Update certification articles
We need to update all of our old certification articles and create new ones for our Back End and Data Visualization certifications.

Here are the current articles:

https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Free-Code-Camp-Front-End-Development-Certificate
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Free-Code-Camp-Full-Stack-Development-Certificate

The new articles can follow the pattern shown in the Front End Development Certificate article. 

Not that these should be named "certification" (certificate is the actual piece of paper).

All the requirements for the certifications are listed in this challenge: http://www.freecodecamp.com/challenges/learn-how-free-code-camp-works

### PR DESCRIPTION
**Pull request by varunu28**
Updated the certification articles for both Frontend and Full Stack development certificates

closes #535

### CODE REVIEW

**Comment by raisedadead** [comment]
@varunu28 Thanks for the PR.

Can you please address the below points as per the original issue #535 :
* Rename the articles. i.e. you have rename the `.md` file for these articles

> Not that these should be named "certification" (certificate is the actual piece of paper).

* Change the wording in the content to reflect "certification" to maintain consistency as noted above. I'll comment on the change inline, to make this clearer for you.
* Add the Data Visualization and Back-end certification articles to this PR.
* Finally [Squash your commits](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one).

Addtional steps:
Remember DO NOT mention issue no.s in the PR title or commit messages.
Add `closes #535` in the PR description once all the points are addressed.

If you need any assistance, you can ping us in the [Wiki Chat Room](https://gitter.im/FreeCodeCamp/Wiki).

File: Free-Code-Camp-Front-End-Development-Certificate.md [hunk 1]
```diff
@@ -3,8 +3,10 @@
...
-1. Complete all of our current Intermediate Front End Projects
+2. Complete all of our current Basic Front End Projects
+3. Complete all of our current Intermediate Algorithms
+4. Complete all of our current Intermediate Front End Projects
+5. Complete all of our current Advanced Algorithms
+6. Complete all of our current Advanced Front End Projects
 
-Once you've completed all of the above, you can complete our "Claim your Front End Development Certificate" challenge, which will require you to sign our Academic Honesty Pledge. Then we will instantly issue your verified certificate, which will become accessible from your portfolio page.
\ No newline at end of file
+Once you've completed all of the above, you can complete our "Claim your Front End Development Certificate" challenge, which will require you to sign our Academic Honesty Pledge. Then we will instantly issue your verified certificate, which will become accessible from your portfolio page.
```
Comment (raisedadead) [inline Free-Code-Camp-Front-End-Development-Certificate.md]: 1. Change `Certificate` > `Certification` and elsewhere.
2. Add the link to the challenge to the text `Claim your Front End Development Certificate`

File: Free-Code-Camp-Full-Stack-Development-Certificate.md [hunk 2]
```diff
@@ -4,8 +4,11 @@ In order to participate in our nonprofit projects, you first need to earn our Fu
 
 ### Here's how to earn our verified Full Stack Development Certificate:
 
-1. Complete all our current Algorithm Challenges
-1. Complete all our current Front End Project Challenges
-1. Complete all our current Back End Project Challenges
+1. Earn our current Front End Development Cerificate
```
Comment (raisedadead) [inline Free-Code-Camp-Full-Stack-Development-Certificate.md]: Change `Certificate` > `Certification` and elsewhere.

File: Free-Code-Camp-Full-Stack-Development-Certificate.md [hunk 3]
```diff
@@ -4,8 +4,11 @@ In order to participate in our nonprofit projects, you first need to earn our Fu
...
+2. Earn our current Data Visualization Cerificate
+3. Earn our current Back End Development Cerificate
+4. Complete all our current Greenfield Nonprofit Projects
+5. Complete all our current Legacy Code Nonprofit Projects
 
-Once you've completed all of the above, you can complete our "Claim your Full Stack Development Certificate" challenge, which will require you to sign our Academic Honesty Pledge. Then we will instantly issue your verified certificate, which will become accessible from your portfolio page.
\ No newline at end of file
+
+Once you've completed all of the above, you can complete our "Claim your Full Stack Development Certificate" challenge, which will require you to sign our Academic Honesty Pledge. Then we will instantly issue your verified certificate, which will become accessible from your portfolio page.
```
Comment (raisedadead) [inline Free-Code-Camp-Full-Stack-Development-Certificate.md]: Add the link to the challenge to the text `Claim your Full Stack Development Certificate`

Comment (varunu28) [inline Free-Code-Camp-Full-Stack-Development-Certificate.md]: Can you please provide the link for Full Stack Development Certification. Mine is showing locked

**Comment by varunu28** [comment]
Someone please squash the commits. I am unable to squash the commits

**Comment by raisedadead** [comment]
Closing in favor of #556

**Pull request closed without merging**